frontend/src/pages/Signup.tsx

```diff
@@ -84,81 +86,88 @@ const Signup = () => {
   if (submitted) {
     return <Redirect to={"/thankyou"} />;
   } else {
-    return <>
-      <Helmet>
-        <title>Clothing-chain | Signup user</title>
-        <meta name="description" content="Signup user"/>
-      </Helmet>
-      <Formik
-        initialValues={{
-          name: "",
-          email: "",
-          phoneNumber: "",
-          newsletter: false,
-          actionsNewsletter: false,
-        }}
-        validationSchema={validate}
-        onSubmit={async (v) => onSubmit(v)}
-      >
-        {(formik) => (
-          <ThreeColumnLayout>
-            <div>
-              <img
-                src={AppIcon}
-                alt="SFM logo"
-                width="200"
-                className={classes.image}
-              />
-              <Typography variant="h3" className={classes.pageTitle}>
-                {t("signup")}
-              </Typography>{" "}
-              <p>{chain?.name}</p>
-              <Form>
-                <TextForm
-                  label="Name"
-                  name="name"
-                  type="text"
-                  className={classes.textField}
+    return (
+      <>
+        <Helmet>
+          <title>Clothing-chain | Signup user</title>
+          <meta name="description" content="Signup user" />
+        </Helmet>
+        <Formik
+          initialValues={{
+            name: "",
+            email: "",
+            phoneNumber: "",
+            newsletter: false,
+            actionsNewsletter: false,
+          }}
+          validationSchema={validate}
+          onSubmit={async (v) => onSubmit(v)}
```

Comment:
If you're making this function async, you also need to await onSubmit. I'm pretty sure `onSubmit={(v) => onSubmit(v)}` will also work, returning a promise, try it?